public/app/features/dashboard-scene/serialization/transformSaveModelToScene.ts

```diff
@@ -208,7 +208,13 @@ export function createDashboardSceneFromDashboardModel(oldModel: DashboardModel)
       isLazy: true,
       children: createSceneObjectsForPanels(oldModel.panels),
     }),
-    $timeRange: new SceneTimeRange(oldModel.time),
+    $timeRange: new SceneTimeRange({
+      from: oldModel.time.from,
+      to: oldModel.time.to,
+      fiscalYearStartMonth: oldModel.fiscalYearStartMonth,
```

Comment:
🤔 For some strange reason, even though we get the correct `fiscalYearStartMonth` here, that value isn't reflected when I render the dashboard using scenes.

https://github.com/grafana/grafana/assets/239999/f204a647-f693-4689-b92d-bd0a6eb1762d

Reply:
@axelavargas good catch! I only checked that it was reflected in the graph time ranges not in the time picker fiscal year selector itself 

Fixed here: 
https://github.com/grafana/scenes/pull/386